stripe/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -501,5 +502,26 @@ data class ConfirmPaymentIntentParams internal constructor(
                 shipping = shipping
             )
         }
+
+        @JvmStatic
+        internal fun createAlipay(
+            context: Context,
+            clientSecret: String,
+            returnUrl: String
+        ): ConfirmPaymentIntentParams {
+            val packageName = context.packageName
+            // TODO(smaskell-stripe): Confirm error handling
+            val version = kotlin.runCatching {
+                context.packageManager.getPackageInfo(packageName, 0).versionName
+            }.getOrDefault("1.0")
```

Comment:
Maybe overkill, but can we abstract this logic into some other class and use it here and `FingerprintRequestParamsFactory`?

Reply:
sure. Also, I was thinking about it and perhaps the runCatching is just overcomplicating things? There is no reason this should ever fail because it's impossible for the current package to not be installed

Reply:
You're probably right, the null checks may be cargo culted.